dayOne/Numbers.java

```diff
@@ -0,0 +1,74 @@
+package dayOne;
+
+import java.util.Scanner;
+
+public class Numbers {
+	public static boolean checkPrime(int inputNumber) {
+		if(inputNumber<=1)
+			return false;
+		if(inputNumber==2||inputNumber==3)
+			return true;
+		if(inputNumber%2==0||inputNumber%3==0)
+			return false;
+		for(int i=5;i*i<=inputNumber/2;i++)
```

Comment:
Can you update this logic and commit your changes?

Reply:
I have updated the code and commited it